app/src/ui/preferences/appearance.tsx

```diff
@@ -118,15 +132,41 @@ export class Appearance extends React.Component<
     return (
       <DialogContent>
         <h2 id="theme-heading">Theme</h2>
-
-        <RadioGroup<ApplicationTheme>
-          ariaLabelledBy="theme-heading"
-          className="theme-selector"
-          selectedKey={selectedTheme}
-          radioButtonKeys={themes}
-          onSelectionChanged={this.onSelectedThemeChanged}
-          renderRadioButtonLabelContents={this.renderThemeSwatch}
-        />
+        <Row>
+          <RadioGroup<ApplicationTheme>
+            ariaLabelledBy="theme-heading"
+            className="theme-selector"
+            selectedKey={selectedTheme}
+            radioButtonKeys={themes}
+            onSelectionChanged={this.onSelectedThemeChanged}
+            renderRadioButtonLabelContents={this.renderThemeSwatch}
+          />
+        </Row>
+
+        <div className="advanced-section">
+          <h2>Title bar style</h2>
+
+          <Select
+            value={this.state.titleBarStyle}
+            onChange={this.onSelectChanged}
+          >
+            <option value="native">Native</option>
+            <option value="custom">Custom</option>
+          </Select>
+
+          <div className="git-settings-description">
+            <ul>
+              <li>
+                <b>Native</b>: Uses the menu system and chrome provided by your
```

Comment:
:art: I was thinking that we'd not show both options here, and instead show the description associated with the selected value. This would allow us to avoid the `<b>` text here.

Rough patch of changes:

```diff
diff --git a/app/src/ui/preferences/appearance.tsx b/app/src/ui/preferences/appearance.tsx
index f7b15caf2..2990bbad3 100644
--- a/app/src/ui/preferences/appearance.tsx
+++ b/app/src/ui/preferences/appearance.tsx
@@ -23,6 +23,15 @@ interface IAppearanceState {
   readonly titleBarStyle: TitleBarStyle
 }
 
+function getTitleDescription(titleBarStyle: TitleBarStyle): string {
+  switch (titleBarStyle) {
+    case 'custom':
+      return 'Uses the menu system and chrome provided by your window manager.'
+    case 'native':
+      return 'Uses the menu system provided by GitHub Desktop, hiding the default chrome provided by 
your window manager.'
+  }
+}
+
 export class Appearance extends React.Component<
   IAppearanceProps,
   IAppearanceState
@@ -129,6 +138,8 @@ export class Appearance extends React.Component<
       ...(supportsSystemThemeChanges() ? [ApplicationTheme.System] : []),
     ]
 
+    const description = getTitleDescription(this.state.titleBarStyle)
+
     return (
       <DialogContent>
         <h2 id="theme-heading">Theme</h2>
@@ -155,16 +166,7 @@ export class Appearance extends React.Component<
           </Select>
 
           <div className="git-settings-description">
-            <ul>
-              <li>
-                <b>Native</b>: Uses the menu system and chrome provided by your
-                window manager.
-              </li>
-              <li>
-                <b>Custom</b>: Uses the menu system provided by GitHub Desktop,
-                hiding the default chrome provided by your window manager.
-              </li>
-            </ul>
+            <span>{description}</span>
           </div>
         </div>
       </DialogContent>
```

Screenshot:

![image](https://github.com/shiftkey/desktop/assets/359239/762529e5-1d96-4525-8d58-6ffb97748d6f)

Reply:
It will be clearer to let user know about different options available before they pick one.

Also it might not be noticeable that the description text will change based on the current selected value.

Reply:
> It will be clearer to let user know about different options available before they pick one.

I'm not sold on this still, given other examples from the settings page stick to one-line summaries (or omit them).

![image](https://github.com/shiftkey/desktop/assets/359239/5fd56f84-bcfd-4478-ad4b-00ec0cb99afc)

![image](https://github.com/shiftkey/desktop/assets/359239/b4235aff-3ff2-45a4-a216-dfe7f4fe15cd)

![image](https://github.com/shiftkey/desktop/assets/359239/804a3285-77d1-44e1-859d-a5c4cb9a1523)

![image](https://github.com/shiftkey/desktop/assets/359239/3a8c2a32-d097-4883-af06-9c4b331a658a)

Given how easy it is to switch between the two, and aside from having to restart it's a low-risk change to toggle, I lean towards keeping the settings page as simple as possible.

> Also it might not be noticeable that the description text will change based on the current selected value.

Eh, I still lean towards simplicity here. But we can iterate on this based on feedback - I just want us to follow the upstream project as much as possible to be consistent...